server/router_utils.go

```diff
@@ -197,29 +197,34 @@ func (server *HTTPServer) readClusterRuleUserParams(
 	return clusterID, ruleID, userID, true
 }
 
-// readClusterRuleParams gets cluster_name and rule_id from current request
+// readClusterRuleParams gets cluster_name, rule_id and error_key from current request
 func (server *HTTPServer) readClusterRuleParams(
 	writer http.ResponseWriter, request *http.Request,
-) (types.ClusterName, types.RuleID, bool) {
+) (types.ClusterName, types.RuleID, types.ErrorKey, bool) {
 	clusterID, successful := readClusterName(writer, request)
 	if !successful {
-		return "", "", false
+		return "", "", "", false
```

Comment:
the right time to use named return values, WDYT?

Reply:
you are right, new commit incoming